diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/CompactInitialization.kt

```diff
@@ -52,7 +52,7 @@ class CompactInitialization(private val configRules: List<RulesConfig>) : Rule("
      * Check property's initializer: if it is a method call, we find all consecutive statements that are this property's
      * fields accessors and wrap them in an `apply` function.
      */
-    @Suppress("UnsafeCallOnNullableType")
+    @Suppress("UnsafeCallOnNullableType", "COMMENTED_OUT_CODE")
```

Comment:
Why? Is this false positive?

Reply:
Because there is comment with `=`

Reply:
And this thing is parsed correctly? 
```
statements like `name.field = value` where name == propertyName
```
 ?